src/components/flatSelect/examples/FlatSelectExamples.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import { IFlatSelectOptionProps } from '../FlatSelectOption';
+import { UUID } from '../../../utils/UUID';
+import { FlatSelectConnected } from '../FlatSelectConnected';
+import { ISvgProps, Svg } from '../../svg/Svg';
+import { IContentProps } from '../../content/Content';
+
+export class FlatSelectExamples extends React.Component<any, void> {
+
+  render() {
+    const defaultFlatSelectOption: IFlatSelectOptionProps[] = [
+      {
+        option: { content: 'Option 1' },
+      }, {
+        option: { content: 'Option 2' },
+      }, {
+        option: { content: 'Option 3' },
+      },
+    ];
+
+    const svg: ISvgProps = {
+      svgName: 'domain-google',
+      svgClass: 'icon',
+    };
+    const getPrepend: IContentProps = { content: () => <Svg {...svg} />, classes: ['mr1'] };
+    const getAppend: IContentProps = { content: () => <Svg {...svg} />, classes: ['ml1'] };
+
+    return (
+      <div className='mt2'>
+        <h1 className='text-blue mb1'>Flat Select List</h1>
+
+        <div className='form-group'>
+          <label className='form-control-label'>Default Flat Select</label>
+          <div className='form-control'>
+            <FlatSelectConnected {...{
+              id: UUID.generate(),
```

Comment:
FlatSelectConnected should probably generate it's own id when added to the store (in your reducer)

Reply:
I disagree, especially if you compose a component out of other components, it's way better to have ids set as prop if you want to know which component are linked together. Suppose you have a table with a filter, and want to do modification to the tableState based on a filterAction, you must know which id this filter has in the store to perform state modification only for this filter and not other ones, and I think you can only have that by having control over the ids of the components.

Reply:
You're absolutely right. Disregard my comment